File: PRAK405-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main() {
    int baris, kelipatan, total = 0;

    printf("Input: ");
    scanf("%d %d", &baris, &kelipatan);

    for (int i = 1; i <= baris; i++) {
        int jumlahBaris = 0;
        
        printf("(");
        for (int j = i; j >= 1; j--) { 
            int nilai = j * kelipatan;
            jumlahBaris += nilai;
            printf("%d * %d", j, kelipatan);  
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(") = %d\n", jumlahBaris);  
        total += jumlahBaris;             
    }
    printf("%d\n", total);
}

File: PRAK405-2410817310009-Muhammad_Ghazi_Rakhmadi.py
baris, kelipatan = map(int, input("Input: ").split())
total = 0  

for i in range(1, baris + 1):
    jumlah_baris = 0  
    print("(", end="")

    for j in range(i, 0, -1):  
        nilai = j * kelipatan
        jumlah_baris += nilai
        print(f"{j} * {kelipatan}", end="")
        if j > 1:
            print(" + ", end="")
    print(f") = {jumlah_baris}")
    total += jumlah_baris
print(total)